viewer/packages/pointclouds/src/potree-three-loader/rendering/shaders/pointcloud.vert

```diff
@@ -350,6 +350,12 @@ float getIntensity() {
 	return w;
 }
 
+vec3 getRgbWithIntensityFallback() {
```

Comment:
What is the reason we want intensity as fallback? It's not always available either, as far as I know, and I've seen some confusion earlier as to what range it lies in, but not sure if that's still a real issue.

Reply:
Just a last resort. There are quite a few cases where RGB isn't available, and in almost all of these intensity will be available I think. Range might be an issue, but it improves situation slightly I guess.